### PR TITLE
feat: ldio naming cleanup

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldi-core/ldes-client/tree-node-supplier/pom.xml [hunk 1]
```diff
@@ -69,6 +69,10 @@
             <artifactId>HikariCP</artifactId>
             <version>${hikari.version}</version>
         </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+            <artifactId>ldi-common</artifactId>
+        </dependency>
```
Comment (jobulcke) [inline ldi-core/ldes-client/tree-node-supplier/pom.xml:75]: Depedency is already already added at lines 14-18

File: ldi-nifi/ldi-nifi-processors/ldes-client-processor/pom.xml [hunk 2]
```diff
@@ -42,6 +42,10 @@
             <version>${testcontainers-postgresql.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+            <artifactId>ldi-common</artifactId>
+        </dependency>
```
Comment (jobulcke) [inline ldi-nifi/ldi-nifi-processors/ldes-client-processor/pom.xml:48]: dependency is already added at lines 20-24

File: ldi-orchestrator/ldio-connectors/ldio-http-in-poller/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/HttpInputPoller.java
Comment (jobulcke) [inline ldi-orchestrator/ldio-connectors/ldio-http-in-poller/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/HttpInputPoller.java:1]: Maybe this class should be renamed to be more conform it's `NAME`: `HttpInputPoller `-> `LdioHttpInPoller`

File: ldi-orchestrator/ldio-connectors/ldio-rdf-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioRdfAdapterAutoConfig.java [hunk 3]
```diff
@@ -14,7 +14,7 @@ public class LdioRdfAdapterAutoConfig {
 	public static final String MAX_JSONLD_CACHE_CAPACITY = "max-jsonld-cache-capacity";
 
 	@SuppressWarnings("java:S6830")
-	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldi.RdfAdapter")
+	@Bean("Ldio:RdfAdapter")
 	public LdioAdapterConfigurator ldiHttpOutConfigurator() {
```
Comment (jobulcke) [inline ldi-orchestrator/ldio-connectors/ldio-rdf-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioRdfAdapterAutoConfig.java:18]: This is more outside of the scope of this story, but I would expect this method name to be more like `ldioAdapterConfigurator` or `ldioRdfConfigurator`

File: ldi-orchestrator/ldio-connectors/ldio-ngsiv2-to-ld-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioNgsiv2ToLdAdapterAutoConfig.java [hunk 4]
```diff
@@ -11,7 +11,7 @@
 public class LdioNgsiv2ToLdAdapterAutoConfig {
 
 	@SuppressWarnings("java:S6830")
-	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldi.NgsiV2ToLdAdapter")
+	@Bean("Ldio:NgsiV2ToLdAdapter")
 	public LdioAdapterConfigurator ldiHttpOutConfigurator() {
```
Comment (jobulcke) [inline ldi-orchestrator/ldio-connectors/ldio-ngsiv2-to-ld-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioNgsiv2ToLdAdapterAutoConfig.java:15]: This is more outside of the scope of this story, but I would expect this method name to be more like `ldioAdapterConfigurator`

File: ldi-orchestrator/ldio-connectors/ldio-rml-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioRmlAdapterAutoConfig.java [hunk 5]
```diff
@@ -11,7 +11,7 @@
 public class LdioRmlAdapterAutoConfig {
 
 	@SuppressWarnings("java:S6830")
-	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldi.RmlAdapter")
+	@Bean("Ldio:RmlAdapter")
 	public LdioAdapterConfigurator ldiHttpOutConfigurator() {
```
Comment (jobulcke) [inline ldi-orchestrator/ldio-connectors/ldio-rml-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioRmlAdapterAutoConfig.java:15]: This is more outside of the scope of this story, but I would expect this method name to be more like `ldioAdapterConfigurator` or `ldioRmlAdapterProcessorConfigurator`

**Review by jobulcke — APPROVED** [review]

File: ldi-nifi/ldi-nifi-processors/ldes-client-processor/pom.xml (hunk 2 above)
Comment (Yalz) [reply ldi-nifi/ldi-nifi-processors/ldes-client-processor/pom.xml:48]: was a rebase thing that i got from PJ's commit .. will remove thanks

File: ldi-core/ldes-client/tree-node-supplier/pom.xml (hunk 1 above)
Comment (Yalz) [reply ldi-core/ldes-client/tree-node-supplier/pom.xml:75]: was a rebase thing that i got from PJ's commit .. will remove thanks

**Pull request merged**